modules/rostests/apitests/kernel32/PrivMoveFileIdentityW.c

```diff
@@ -60,6 +60,7 @@ TestPrivMoveFileIdentityW(VOID)
     HANDLE hDest;
     WCHAR Self[MAX_PATH];
     OFSTRUCT ReOpen;
+    BOOL ret;
```

Comment:
Arguably `Ret` suits the surrounding style here more as well.

Reply:
Done.